src/applications/gqrx/receiver.cpp

```diff
@@ -124,14 +124,7 @@ receiver::receiver(const std::string input_device,
 
     audio_udp_sink = make_udp_sink_f();
 
-#ifdef WITH_PULSEAUDIO
-    audio_snk = make_pa_sink(audio_device, d_audio_rate, "GQRX", "Audio output");
-#elif WITH_PORTAUDIO
-    audio_snk = make_portaudio_sink(audio_device, d_audio_rate, "GQRX", "Audio output");
-#else
-    audio_snk = gr::audio::sink::make(d_audio_rate, audio_device, true);
-#endif
-
+    make_audio_device(audio_device);
```

Comment:
I think this would be clearer and more consistent with the rest of the code if `make_audio_device` returned the device instead of internally assigning it to `audio_snk`.

Reply:
> `make_audio_device`

Just a bad name selection.
Renamed to `create_audio_device`. I hope, this should not interfere with GNU Radio naming conventions.
This function was created to eliminate some common code and get rid of some `#ifdefs`. Making it return the device would add more `#ifdefs`.

Reply:
> Making it return the device would add more `#ifdefs`.

I suppose it would. I guess this is fine as it is.